From 9dd46e9271f883dbcfac0a00857262a1f36a2629 Mon Sep 17 00:00:00 2001
From: rajugajjela <104496293+rajugajjela@users.noreply.github.com>
Date: Tue, 16 Aug 2022 11:26:33 +0530
Subject: [PATCH 09/28] Delete D4

---
 D1/D4 | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 D1/D4

diff --git a/D1/D4 b/D1/D4
deleted file mode 100644
index 8b13789..0000000
--- a/D1/D4
+++ /dev/null
@@ -1 +0,0 @@
-
-- 
2.25.1

