From 756ed5436d924af697eb15ab6d40bacb5a75e005 Mon Sep 17 00:00:00 2001
From: rajugajjela <104496293+rajugajjela@users.noreply.github.com>
Date: Tue, 16 Aug 2022 11:56:18 +0530
Subject: [PATCH 18/28] Delete D3 directory

---
 D3/D5/f4 | 1 -
 D3/f4    | 1 -
 2 files changed, 2 deletions(-)
 delete mode 100644 D3/D5/f4
 delete mode 100644 D3/f4

diff --git a/D3/D5/f4 b/D3/D5/f4
deleted file mode 100644
index 8b13789..0000000
--- a/D3/D5/f4
+++ /dev/null
@@ -1 +0,0 @@
-
diff --git a/D3/f4 b/D3/f4
deleted file mode 100644
index 8b13789..0000000
--- a/D3/f4
+++ /dev/null
@@ -1 +0,0 @@
-
-- 
2.25.1

