From 137a46513186b6b915d0c6d872f17bc3f3d43fdd Mon Sep 17 00:00:00 2001
From: rajugajjela <104496293+rajugajjela@users.noreply.github.com>
Date: Tue, 16 Aug 2022 11:56:41 +0530
Subject: [PATCH 20/28] Delete D2 directory

---
 D2/f2 | 5 -----
 D2/f3 | 1 -
 2 files changed, 6 deletions(-)
 delete mode 100644 D2/f2
 delete mode 100644 D2/f3

diff --git a/D2/f2 b/D2/f2
deleted file mode 100644
index 2f153e1..0000000
--- a/D2/f2
+++ /dev/null
@@ -1,5 +0,0 @@
-LOCAL_PRIVILEGED_MODULE := true
-
-LOCAL_MODULE_SUFFIX := .apk
-
-LOCAL_SRC_FILES     := MS21.0.8.apk
diff --git a/D2/f3 b/D2/f3
deleted file mode 100644
index 8b13789..0000000
--- a/D2/f3
+++ /dev/null
@@ -1 +0,0 @@
-
-- 
2.25.1

