From 68e8419678410b54e56dc67e79aeb31c04386927 Mon Sep 17 00:00:00 2001
From: rajugajjela <rgajjela@innominds.com>
Date: Mon, 29 Aug 2022 16:12:54 +0530
Subject: [PATCH 22/28] updated

---
 nani.txt | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 nani.txt

diff --git a/nani.txt b/nani.txt
new file mode 100644
index 0000000..a38cb6d
--- /dev/null
+++ b/nani.txt
@@ -0,0 +1,2 @@
+hiii nani
+hloo "how are you"
-- 
2.25.1

